docs/examples/ds-ig.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ig
+  labels:
+    k8s-app: example-ig
+spec:
+  selector:
+    matchLabels:
+      name: example-ig
+  template:
+    metadata:
+      labels:
+        name: example-ig
+    spec:
+      containers:
+      - name: ig
+        # Replace this with your own image
+        image: alpine:3.17
```

Comment:
Why not to use the ig image directly or mention to use the image prepared in the `Using ig in a custom Kubernetes pod` section?

Reply:
Mmmmm I suppose it is because this example is about "how to add the capability of using `ig` from your own application", so you need to include the binary in an already existing app. Instead, using directly the ig image would be like "how to easily run `ig` on Kubernetes". It slightly different. We could have this example with the ig image by default but mention in the documentation to run [this](https://github.com/inspektor-gadget/inspektor-gadget/blob/a9c054e4a699890f75ad43b3add65cfcbf886c42/docs/examples/ds-ig.yaml#L24-L29) in case they want to use their own base image.

Reply:
> Why not to use the ig image directly or mention to use the image prepared in the Using ig in a custom Kubernetes pod section?

That image is not published and I wanted to show an example that an user can just use to try it out. We can't easily use the default `ig` image because it does not provide the `sleep inf` binary. We could write directly the command `ig trace exec` in the yaml, but that's a bit weird on a DaemonSet with multiple nodes because we can't directly select the node where we want to run the tracer.

To avoid confusion, I could write 2 different sections:
- how to use the existing ig container image in a Kubernetes pod. The gadget `ig trace exec` would be hard coded in the pod yaml because we don't have `sleep inf`.
- how to write your own container image with `ig`. I can give the Dockerfile and a DaemonSet with `sleep inf` and the kubectl-exec workflow.